docker-compose.yaml

```diff
@@ -73,33 +79,52 @@ services:
     networks:
     - web
     volumes:
-    - /etc/localtime:/etc/localtime:ro
+    - /dev/rtc:/dev/rtc:ro
     - ./media/couchpotato/movies:/movies
     - ./media/couchpotato/config/:/config
     - ./media/transmission/downloads/:/downloads
     environment:
-    - PUID="${UID}"
-    - PGID="${GID}"
+    - "PUID=${USER_ID}"
+    - "PGID=${GROUP_ID}"
     labels:
+    - "traefik.enable=true"
     - "traefik.backend=couchpotato"
     - "traefik.frontend.rule=Host:couchpotato.${DOMAIN_NAME}"
     - "traefik.port=5050"
+    - "traefik.docker.network=web"
     depends_on:
     - transmission
 
-  # uncomment to enable torznab as a T411 proxy for couchpotato and sonarr
+  # uncomment to enable torznab as a T411 proxy for sonarr
   # torznab's image will need to be built
   t411proxy:
```

Comment:
I think it's something we should prompt our user for in the future setup script but yeah, I'll comment it out.